<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>registraor_window</class>
    <widget class="QWidget" name="registraor_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
       <width>257</width>
       <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
        <widget class="QToolBox" name="toolBox">
   <property name="geometry">
    <rect>
        <x>10</x>
        <y>10</y>
        <width>231</width>
        <height>391</height>
    </rect>
   </property>
            <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                </sizepolicy>
            </property>
            <property name="currentIndex">
                <number>3</number>
            </property>
            <widget class="QWidget" name="page_2">
                <property name="geometry">
                    <rect>
                        <x>0</x>
                        <y>0</y>
                        <width>231</width>
                        <height>256</height>
                    </rect>
                </property>
                <attribute name="label">
                    <string>Добавить пациента</string>
                </attribute>
                <widget class="QWidget" name="verticalLayoutWidget">
                    <property name="geometry">
                        <rect>
                            <x>10</x>
                            <y>30</y>
                            <width>216</width>
                            <height>201</height>
                        </rect>
                    </property>
                    <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0,0,0">
                        <property name="sizeConstraint">
                            <enum>QLayout::SetDefaultConstraint</enum>
                        </property>
                        <item row="1" column="0" colspan="2">
                            <widget class="QLineEdit" name="user_password_text_box">
                                <property name="echoMode">
                                    <enum>QLineEdit::Password</enum>
                                </property>
                                <property name="placeholderText">
                                    <string>Пароль</string>
                                </property>
                            </widget>
                        </item>
                        <item row="5" column="0" alignment="Qt::AlignLeft">
                            <widget class="QRadioButton" name="user_men_radio">
                                <property name="text">
                                    <string>муж</string>
                                </property>
                                <property name="checked">
                                    <bool>true</bool>
                                </property>
                            </widget>
                        </item>
                        <item row="3" column="0" colspan="2">
                            <widget class="QLabel" name="label_4">
                                <property name="text">
                                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Дата
                                        рождения&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                                    </string>
                                </property>
                            </widget>
                        </item>
                        <item row="2" column="0" colspan="2">
                            <widget class="QLineEdit" name="user_fio_text_box">
                                <property name="text">
                                    <string/>
                                </property>
                                <property name="placeholderText">
                                    <string>Фамилия Имя Отчество</string>
                                </property>
                            </widget>
                        </item>
                        <item row="4" column="0" colspan="2">
                            <widget class="QDateEdit" name="user_birthday"/>
                        </item>
                        <item row="6" column="0" colspan="2">
                            <widget class="QLineEdit" name="user_zip_text_box">
                                <property name="placeholderText">
                                    <string>Индекс больницы</string>
                                </property>
                            </widget>
                        </item>
                        <item row="0" column="0" colspan="2">
                            <widget class="QLineEdit" name="user_login_text_box">
                                <property name="placeholderText">
                                    <string>Логин</string>
                                </property>
                            </widget>
                        </item>
                        <item row="5" column="1">
                            <widget class="QRadioButton" name="user_women_radio">
                                <property name="text">
                                    <string>жен</string>
                                </property>
                            </widget>
                        </item>
                        <item row="8" column="0" colspan="2">
                            <widget class="QPushButton" name="add_patient_button">
                                <property name="text">
                                    <string>Добавить пациента</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </widget>
            <widget class="QWidget" name="page_3">
                <property name="geometry">
                    <rect>
                        <x>0</x>
                        <y>0</y>
                        <width>231</width>
                        <height>256</height>
                    </rect>
                </property>
                <attribute name="label">
                    <string>Добавить врача</string>
                </attribute>
                <widget class="QWidget" name="gridLayoutWidget_2">
                    <property name="geometry">
                        <rect>
                            <x>10</x>
                            <y>0</y>
                            <width>211</width>
                            <height>257</height>
                        </rect>
                    </property>
                    <layout class="QGridLayout" name="gridLayout_3">
                        <item row="2" column="0">
                            <widget class="QLineEdit" name="add_fio_doctor_text_box">
                                <property name="placeholderText">
                                    <string>ФИО врача</string>
                                </property>
                            </widget>
                        </item>
                        <item row="7" column="0">
                            <widget class="QTextEdit" name="add_discription_doctor_text_box">
                                <property name="placeholderText">
                                    <string>Дополнительное описание</string>
                                </property>
                            </widget>
                        </item>
                        <item row="5" column="0">
                            <widget class="QLabel" name="label_5">
                                <property name="text">
                                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Дата
                                        рождения&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                                    </string>
                                </property>
                            </widget>
                        </item>
                        <item row="4" column="0">
                            <widget class="QLineEdit" name="add_password_doctor_text_box">
                                <property name="placeholderText">
                                    <string>Пароль</string>
                                </property>
                            </widget>
                        </item>
                        <item row="6" column="0">
                            <widget class="QDateEdit" name="add_date_birthday"/>
                        </item>
                        <item row="3" column="0">
                            <widget class="QLineEdit" name="add_login_doctor_text_box">
                                <property name="placeholderText">
                                    <string>Логин</string>
                                </property>
                            </widget>
                        </item>
                        <item row="11" column="0">
                            <widget class="QPushButton" name="create_doctor_button">
                                <property name="text">
                                    <string>Добавить врача</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </widget>
            <widget class="QWidget" name="page">
                <property name="geometry">
                    <rect>
                        <x>0</x>
                        <y>0</y>
                        <width>231</width>
                        <height>256</height>
                    </rect>
                </property>
                <attribute name="label">
                    <string>Нанять врача</string>
                </attribute>
                <widget class="QWidget" name="gridLayoutWidget_3">
                    <property name="geometry">
                        <rect>
                            <x>10</x>
                            <y>0</y>
                            <width>211</width>
                            <height>241</height>
                        </rect>
                    </property>
                    <layout class="QGridLayout" name="gridLayout_4">
                        <item row="7" column="0">
                            <widget class="QLabel" name="label_6">
                                <property name="text">
                                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Дата
                                        начала
                                        работы&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                                    </string>
                                </property>
                            </widget>
                        </item>
                        <item row="13" column="0">
                            <widget class="QPushButton" name="take_button">
                                <property name="text">
                                    <string>Добавить</string>
                                </property>
                            </widget>
                        </item>
                        <item row="12" column="0">
                            <widget class="QTimeEdit" name="take_doctor_time_end">
                                <property name="displayFormat">
                                    <string>HH:mm</string>
                                </property>
                            </widget>
                        </item>
                        <item row="4" column="0">
                            <widget class="QLineEdit" name="take_zip_doctor_text_box">
                                <property name="placeholderText">
                                    <string>Индекс больницы</string>
                                </property>
                            </widget>
                        </item>
                        <item row="9" column="0">
                            <widget class="QLabel" name="label_8">
                                <property name="text">
                                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Время
                                        начала
                                        смены&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                                    </string>
                                </property>
                            </widget>
                        </item>
                        <item row="10" column="0">
                            <widget class="QTimeEdit" name="take_doctor_time_begin">
                                <property name="displayFormat">
                                    <string>HH:mm</string>
                                </property>
                            </widget>
                        </item>
                        <item row="5" column="0">
                            <widget class="QLineEdit" name="take_special_doctor_text_box">
                                <property name="text">
                                    <string/>
                                </property>
                                <property name="placeholderText">
                                    <string>Назавание специальности</string>
                                </property>
                            </widget>
                        </item>
                        <item row="11" column="0">
                            <widget class="QLabel" name="label_9">
                                <property name="text">
                                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Время
                                        конца
                                        смены&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                                    </string>
                                </property>
                            </widget>
                        </item>
                        <item row="3" column="0">
                            <widget class="QLineEdit" name="take_login_doctor_text_box">
                                <property name="placeholderText">
                                    <string>Логин врача</string>
                                </property>
                            </widget>
                        </item>
                        <item row="8" column="0">
                            <widget class="QDateEdit" name="take_doctor_date_being_work"/>
                        </item>
                    </layout>
                </widget>
            </widget>
            <widget class="QWidget" name="page_4">
                <property name="geometry">
                    <rect>
                        <x>0</x>
                        <y>0</y>
                        <width>231</width>
                        <height>256</height>
                    </rect>
                </property>
                <attribute name="label">
                    <string>Сменить больницу пользователя</string>
                </attribute>
                <widget class="QWidget" name="gridLayoutWidget">
                    <property name="geometry">
                        <rect>
                            <x>10</x>
                            <y>50</y>
                            <width>211</width>
                            <height>91</height>
                        </rect>
                    </property>
                    <layout class="QGridLayout" name="gridLayout_2">
                        <item row="3" column="0">
                            <widget class="QLineEdit" name="change_zip_text_box">
                                <property name="placeholderText">
                                    <string>Новый индекс больницы</string>
                                </property>
                            </widget>
                        </item>
                        <item row="2" column="0">
                            <widget class="QLineEdit" name="change_login_text_box">
                                <property name="placeholderText">
                                    <string>login пользователя</string>
                                </property>
                            </widget>
                        </item>
                        <item row="4" column="0">
                            <widget class="QPushButton" name="change_hospital_user_button">
                                <property name="text">
                                    <string>Сменить привязку к больницы</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </widget>
            <widget class="QWidget" name="page_5">
                <property name="geometry">
                    <rect>
                        <x>0</x>
                        <y>0</y>
                        <width>231</width>
                        <height>256</height>
                    </rect>
                </property>
                <attribute name="label">
                    <string>Уволить врача из больницы</string>
                </attribute>
                <widget class="QWidget" name="gridLayoutWidget_4">
                    <property name="geometry">
                        <rect>
                            <x>10</x>
                            <y>70</y>
                            <width>191</width>
                            <height>101</height>
                        </rect>
                    </property>
                    <layout class="QGridLayout" name="tet">
                        <item row="2" column="0">
                            <widget class="QPushButton" name="del_button">
                                <property name="text">
                                    <string>Уволить</string>
                                </property>
                            </widget>
                        </item>
                        <item row="0" column="0">
                            <widget class="QLineEdit" name="del_login_text_box">
                                <property name="placeholderText">
                                    <string>Логин врача</string>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="0">
                            <widget class="QLineEdit" name="del_password_text_box">
                                <property name="placeholderText">
                                    <string>Индекс больницы</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
